third_party/libunwindstack/include/unwindstack/Maps.h

```diff
@@ -70,6 +70,15 @@ class Maps {
   const_iterator begin() const { return maps_.begin(); }
   const_iterator end() const { return maps_.end(); }
 
+  // Inserts a new MapInfo before pos and returns an iterator pointing to the inserted MapInfo.
+  iterator Insert(iterator pos, uint64_t start, uint64_t end, uint64_t offset, uint64_t flags,
+                  const std::string& name);
+  iterator Insert(iterator pos, uint64_t start, uint64_t end, uint64_t offset, uint64_t flags,
+                  const std::string& name, uint64_t load_bias);
+
+  // Removes the MapInfo at pos and returns the iterator following the removed MapInfo.
+  iterator erase(const_iterator pos);
```

Comment:
`Erase`?

Reply:
Hehe, the reasoning was that this has the same signature as one of the `erase` methods of an STL container, while `Insert` doesn't. Any thought?

Reply:
SGTM. I suspected that but wanted to ask anyway.